### PR TITLE
Adds default heights to slider and calculator

### PR DESCRIPTION
**Pull request by eireland**
Fixes height and width for newly opened sliders and calculators to a default value

### CODE REVIEW

**Review by kswenson — CHANGES_REQUESTED** [review]
I removed these defaults intentionally. For fixed-size components, the sizes should be defined by the component, not by the registration. If there's something you're trying to achieve here we should talk about it.

**Review by bfinzer — APPROVED** [review]
I defer to Kirk wrt default sizes

File: v3/src/components/slider/slider-registration.ts [hunk 1]
```diff
@@ -44,7 +44,10 @@ registerTileComponentInfo({
   tileEltClass: kSliderTileClass,
   Icon: SliderIcon,
   defaultWidth: 300,
-  isFixedHeight: true
+  isFixedHeight: true,
+  // must be in sync with rendered size for auto placement code
+    defaultHeight: 73
```
Comment (kswenson) [inline v3/src/components/slider/slider-registration.ts:49]:
```suggestion
  defaultHeight: 73
```

**Review by kswenson — APPROVED** [review]
👍 @eireland and I looked at this together and determined that it is appropriate for the `defaultWidth` and `defaultHeight` to be specified in registration for fixed-size components because the auto-placement code requires it. Instead, the `FreeTileComponent` no longer imposes a dynamic width/height on fixed-size components.

**Pull request merged**